CoreFeatures/FeatureManagement/DownloadFeatureActor/src/main/java/info/smart_tools/smartactors/feature_management/download_feature_actor/DownloadFeatureActor.java

```diff
@@ -31,15 +33,30 @@
     private final IFieldName repositoryTypeFN;
     private final IFieldName repositoryUrlFN;
 
+    private final static String DOWNLOAD_DIRECTORY = "downloads";
+    private final static String IOC_FEATURE_REPOSITORY_STORAGE_NAME = "feature-repositories";
+    private final static String MAVEN_ARTIFACT_SCOPE = "runtime";
+    private final static String ARCHIVE_POSTFIX = "archive";
+
+    //TODO: this parameters would be took out into the config.json as actor arguments
+    private static final String DEF_PACKAGE_TYPE = "jar";
+    private final static String FIELD_NAME_FACTORY_STARTEGY_NAME =
+            "info.smart_tools.smartactors.iobject.ifield_name.IFieldName";
+    private final static List<String> FILE_TYPE_LIST = new ArrayList<String>() {{
```

Comment:
Why do you use inner classes while you are so strongly against inner classes?

In this particular case `Arrays.asList(...)` should be preferred (or `new ArrayList(Arrays.asList(...))` is list should be mutable).

Reply:
Agree. I planned remove this code on next refactoring iteration. But if you want I'll change this code before refactoring.

Reply:
MASSAGE_... constants should be replaced by MESSAGE_... ones